system_hooks/src/contract_deployer.rs

```diff
@@ -192,3 +198,21 @@ where
         _ => Ok(Err("Contract deployer hook: unknown selector")),
     }
 }
+
+///
+/// We add some ergs cost to account for work charged in native only.
```

Comment:
Can we include it during `set_bytecode_details` call?

Reply:
Yes, see line 216